README.adoc

```diff
@@ -157,18 +157,18 @@ have IPv4 addresses, the address on eth2 is used.
 === HTTP Proxy Settings
 
 In an environment where the HTTP traffic needs to pass through an
-HTTP proxy server, set the `http_proxy`, `http_proxy_user` and
-`http_proxy_password` proxy configurations in the Vagrantfiles to enable
+HTTP proxy server, set the `proxy`, `proxy_user` and
```

Comment:
>  I manually tested it and its working now. Previously, it wasn't.

I take your word for it

> I idea is to run HTTPServer as forward proxy server to a dummy simple http server. 

HTTPServer? How come. I don't understand what you are planning to do. WEBrick::HTTPProxyServer should do imo, but I might be missing a piece.

> Note, that I am using a dummy simple http server as I couldn't find a way for proxy server to always return some fixed content.

Override the service method. AFAICT it contains the request and response instances. Using the response instance you should be able to return whatever you want.

> I was thinking to add the proxy test here but I think we cannot do so for this change.

I don't quite understand why not, but whatever ...

> The implementation we discussed during install-cli proxy feature is different than this feature.

Slightly different sure, but only varies in a small degree. If one can do the discussed test for _install-cli_ I am pretty sure one can test the proxy config settings as well.

OOI what have you tried? Do you have any code? Any particular problem or stack trace?